libs/designsystem/src/lib/components/list/list.component.ts

```diff
@@ -112,6 +105,9 @@ export class ListComponent implements OnInit, AfterViewInit, OnChanges {
   };
 
   private async getVirtualDataset(index: number, count: number): Promise<any> {
+    if (!this.useVirtualScroll) {
+      return Promise.resolve();
```

Comment:
Would it make sense to throw in a `console.warning('You are trying to use: getVirtualDataset but useVirtualScroll is false)` or similar.

Reply:
This check no longer exists as the virtual dataset is referenced directly in "virtual" part of the template. I don't think it would be an easy developer mishap to reference the wrong datasource - so to answer your question, I dont think it makes sense adding a warning at this point ;)